gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem
+@rem Copyright 2015 the original author or authors.

```

Comment:
@BhupiSindhwani - can we update this to 2020 latest template. Example is in the build file.